Integrations/EWSMailSender/EWSMailSender.py

```diff
@@ -138,6 +149,18 @@ def send_email(to, subject, body="", bcc=None, cc=None, replyTo=None, htmlBody=N
         with open(file_path, 'rb') as f:
             attachments.append(FileAttachment(content=f.read(), name=attachment_name))
 
+    for i in range(0, len(file_entries_for_attachments_inline)):
```

Comment:
you are not using `i` (if you do need it, use `enumerate`)
```suggestion
    for entry_id, attachment_name_inline in file_entries_for_attachments_inline:
```

Reply:
Same reason here. If we change to enumerator, it's better to change also in first for loop.
Basically the main reason is not to break backward.

Reply:
breaking backward is not an issue here, since the result is the same.
this why is just more pythonic while the other is the C-way